GregWeng.md

```diff
@@ -1,3 +1,18 @@
+## 08/10 ~ 08/14
+
+LockScreen: fix interrmitent errors on Mulet
+
+* Still tracing the possible root case, and still can't successfully reproduce it on my local machine, no matter on Mac or Ubuntu
+  * As a result I can only debug it on treeheader per half hour (modifying, pushing and collecting results)
+  * Will keep trying to reproduce it on local consoles, but I have no further ideas; may ask Alexandre his best strategy
+
+Performance Tools:
+* Bug 1186762 - [Build][Raptor Transformer] Simplify the command to make Raptor + transforming process (r+)
+  * Set checkin-needed but not get landed yet. Will land it manually later.
+
+mozSettings Performance Tuning:
```

Comment:
@snowmantw This is probably worthy of a bug and some progressive notes as comments too, please talk to Cervantes on that.